app/views/layouts/application.html.erb

```diff
@@ -4,6 +4,14 @@
     <meta name="sentry-dsn" content="<%= ENV["SENTRY_DSN"] %>">
     <meta name="sentry-current-env" content="<%= ENV["SENTRY_CURRENT_ENV"] %>">
   <% end %>
+
+  <% if ENV["GOOGLE_TAG_MANAGER_ID"] && ENV["GOOGLE_TAG_MANAGER_AUTH"] && ENV["GOOGLE_TAG_MANAGER_PREVIEW"] %>
```

Comment:
Are we fine excluding a scenario where we've only got tag_manager_id ? I know it's not that realistic since Andy plans to use all 3 but I'd have thought we'd only need to check that env var and be happy if some of the others are nil.

Reply:
You're right @kevindew, we only need to check for `GOOGLE_TAG_MANAGER_ID`. Updated.